app/assets/javascripts/discourse/app/lib/admin-plugin-header-actions.js

```diff
@@ -0,0 +1,13 @@
+let pluginHeaderActionComponents = [];
```

Comment:
```suggestion
let pluginHeaderActionComponents = {};
```

right? 😛 

(I'd recommend using `Map` though)

Reply:
Ugh stupid, thanks for pointing that out :+1: